src/clj_infrastructure/db.clj

```diff
@@ -653,3 +655,67 @@
         col-value-pairs (partition 2 (rest key-string-parts))
         where-conditions (map (fn [[col value]] (str col "='" value "'")) col-value-pairs)]
     (string/join " and " where-conditions)))
+
+(defn split-statements
+  [sql-text]
+  (let [statements (clojure.string/split sql-text #";")]
```

Comment:
There are semicolons within comments in SQL files used by BEETL, so deployer beware.